R/collaborator_roles.csv

```diff
@@ -1,14 +1,14 @@
-,,README,"The project README can contain contributor guidelines, a link to a project website (e.g. made with GitHub pages), and licence information."
-,,Releases,"Releases mark milestones in the progression of a project (e.g. version of manuscript submitted to a journal). If a repository is linked to Zenodo, releases are automatically archived and given a unique DOI."
-Co-author,"Contributes writing, but has little to no experience with programming. Even if familiar with programming, they are unfamiliar with git and GitHub.",GitHub Pages,The Project Manager can use GitHub pages to share reports and figures with Co-Authors who can access up-to-date information through the project website.
-,,Discussions,Can be used as a forum to discuss proposed changes in a manuscript.
-,,Markdown Editors,Can contribute text as Markdown using a web interface through GitHub (Add File button) or through extensions like HackMD
-Code contributor,Anyone who contributes code to the project.,Pull Requests,Pull Requests are a natural way to contribute code to the project. These changes can be reviewed by the project manager or code reviewers before integrating them.
-Code reviewer,"Anyone asked to review project code (e.g. a co-author or a peer-reviewer). They are familiar with coding, but not necessarily with git or GitHub (but they are willing to learn).",Issues,Reviewers can highlight specific lines of code and create issues reffering to them with suggestions
-,,Pull Requests,Reviewers can either be assigned to review pull requests (e.g. from code contributors) or can make pull requests with recommended changes.
-Community,Anyone who wishes to use or adapt data or code for their own analysis,Fork,Anyone can use this feature to create a linked copy of the repository on their own GitHub account.
-,,Discussions,Community members can be directed to the Discussions tab as a place to ask clarifying questions about data and code.
+Role,Description,GitHub Feature,Use Case
+Project manager,"Owns the GitHub repository for a project, and leads the academic project (e.g., lead author of a manuscript).",Permissions,"The project manager can manage permissions of the repository to give push/pull access to contributors within their organization. They can protect the main branch from push access, if desired, to be sure all contributions are reviewed as pull requests."
+,,Issues,"Project managers can use issues to track tasks and assign them to co-authors and code contributors. Tasks may including writing, code features, and responding to reviewer comments."
+,,README,"The project README can contain contributor guidelines, a link to a project website (e.g. made with GitHub pages), and licence information."
+,,Releases,"Releases mark milestones in the progression of a project (e.g. version of manuscript submitted to a journal). If a repository is linked to Zenodo, releases are automatically archived and given a unique DOI."

```

Comment:
I think "Release" should be one of the definitions in Box 1

>**Release:** At any point a **release** can be made on GitHub to mark a significant milestone in the progression of a repository.  While this GitHub feature is designed with releases of new versions of software in mind, it can also be used to create a snapshot of a repository at significant stages like submission, revision, and acceptance of an associated manuscript.

Reply:
Then the use here can be as simple as "Trigger archival of code and data and DOI assignment by Zenodo or Dryad" (I actually am only sure about Zenodo integration, would need to check how Dryad works).